content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+   :align: center
+   :alt: Syncing Meta WhatsApp templates to the Odoo database, with the 'Sync Templates'
+         highlighted.
+
+A successful message in green should populate in the upper right hand corner with the number of
+templates updated.
+
+Using variables
+---------------
+
+Dynamic variables can be input on the WhatsApp template on the Meta business manager. This process
+should be completed during the initial layout and design through the Meta business console. Dynamic
+variables reference certain fields within the Odoo database to produce unique data in the WhatsApp
+message through the template. Dynamic variables are encoded to display fields from within the
+database referencing fields from within a model.
+
+.. example::
+   Many companies like to customize their WhatsApp messages with a personalized piece of customer
+   information to grab attention. This can be accomplished in Odoo by referencing a field within a
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` -->  field on the :guilabel:`Sales Order` model.
+
+First navigate to  `<https://business.facebook.com/wa/manage/home>`_. Then go to
+:menuselection:`Account tools --> Message templates`. Click on one and then :guilabel:`Edit
+template` or create a new template.
+
+Insert a variable by clicking on the :guilabel:`Add variable` button in the template. Variables can
+be added to the :guilabel:`Header`, :guilabel:`Body`, and :guilabel:`Footer` of the WhatsApp
+template.
+
+.. image:: whatsapp/add-variable.png
+   :align: center
+   :alt: Adding a variable in the body of a WhatsApp template.
+
+.. seealso::
+   :ref:`whatsapp/templates`
+
+After the template has been submitted and approved by Meta, the template needs to be synced with
+Odoo. To sync the template see this documentation: :ref:`whatsapp/sync`.
+
+Once the template(s) has been uploaded to the Odoo database navigate to :menuselection:`WhatsApp app
+--> Templates` and open the template that should be modified. Click :guilabel:`Reset to draft` to
+edit the template. The dynamic variable(s) need to be configured on the Odoo side as well to specify
+the field that should be referenced.
+
+To edit the dynamic variables on a template, first change the :guilabel:`Type` to :guilabel:`Field
+of Model`. This allows Odoo to reference a field within a model to produce unique data in the
+end message.
+
+Next, go to the :guilabel:`Variables` tab and edit the :guilabel:`Field` field of the dynamic
+variables. The :guilabel:`Applies to` field can be edited prior to ensure the correct model and
+field are referenced.
+
+.. tip::
+   By default Meta uses the :guilabel:`Applies to` field set to the :guilabel:`Contact` model.
+
+To search the available fields, simply type in the front-end name (on user-interface) of the field
+in the search. This will find a result from all of the available fields for the model
```

Comment:
```suggestion
in the search box. This will find a result from all of the available fields for the model
```


If it's not a box, change to whatever it is (field, etc)